packages/extension-browser/src/background-script.ts

```diff
@@ -216,8 +216,8 @@ const webRequestEvents = [
 ];
 
 const webRequestHandlers = webRequestEvents.map((event) => {
```

Comment:
Comment: Maybe we shouldn't rely that much in infering the types in this kind of situations 🙄

Reply:
Yeah, that's a fairly complex return value. I agree having an explicit type to describe it would be a good idea. I'll update the PR.